edx/edx-enterprise
# Make discovery-service lookups site-aware

Replace all the hard-coded lookups of the discovery service to
instead lookup the value of COURSE_CATALOG_API_URL from the Enterprise
Customer's site configuration, falling back to
settings.COURSE_CATALOG_API_URL if not specified by the Enterprise
Customer. Also fixes cross-site error in 'Manage Learners' form by using
the base url of the current site to look up enrollment in a course.

WL-1241

**JIRA: https://openedx.atlassian.net/browse/WL-1241

**Testing instructions:
1. Create 2 enterprise customers, each pointing to a different white label site
2. Make sure you can manually enroll users for each enterprise when logged in to business.sandbox.edx.org/admin site
3.  Make sure you can manually enroll users for a given enterprise when logged in to {site-name}.sandbox.edx.org/admin site

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Author concerns:** None
